app/code/Magento/Customer/Model/Url.php

```diff
@@ -256,4 +261,23 @@ private function getRequestReferrer()
         }
         return null;
     }
+
+    /**
+     *  Check if Referrer url is no route url
+     *
+     * @param string $url
+     * @return bool
+     */
+    private function isNoRouteUrl($url)
+    {
+        $defaultNoRouteUrl = $this->scopeConfig->getValue(
+            'web/default/no_route',
```

Comment:
It is better if declare `private const XML_PATH_WEB_DEFAULT_NO_ROUTE = 'web/default/no_route'`